share/shwap/p2p/bitswap/bitswap.go

```diff
@@ -103,13 +121,16 @@ func NewClient(
 ) *client.Client {
 	opts := []client.Option{
 		client.SetSimulateDontHavesOnTimeout(simulateDontHaves),
-		client.ProviderSearchDelay(providerSearchDelay),
-		client.RebroadcastDelay(delay.Fixed(rebroadcastDelay)),
+		client.WithDontHaveTimeoutConfig(simulateDontHaveConfig),
+		client.WithDisabledMessageQueueRebroadcast(disablePerPeerRetries),
 		// Prevents Has calls to Blockstore for metric that counts duplicates
 		// Unnecessary for our use case, so we can save some disk lookups.
 		client.WithoutDuplicatedBlockStats(),
-		client.WithDontHaveTimeoutConfig(dontHaveTimeoutConfig),
-		client.WithDisabledMessageQueueRebroadcast(true),
+
+		// These two options have mixed up named. One should be another and vice versa.
+		// TODO(@Wondertan): Make an issue
+		client.ProviderSearchDelay(broadcastDelay),
+		client.RebroadcastDelay(delay.Fixed(provSearchDelay)),
```

Comment:
This is super confusing, why can't you just fix it in this PR?

Reply:
This has to be fixed in Bitswap itself. They literally mixed up value.

* `WithPendussyFilled` fills renussy 
* `WithRenussyFilled`  fills pendussy

Reply:
Oh god 🤦🏻

Reply:
![image](https://github.com/user-attachments/assets/c5b52fe8-8a39-444e-a3ab-e54ffab47336)

Reply:
The shit I had to go through guys to even realize that...

Reply:
Ah, thats an old meme of ours ;)